### DISCUSSION TITLE
Baremetal migration.

### DISCUSSION THREAD

**Discussion by blakjak44** [rook/rook#12045] [Q&A] [answered]
We have a Ceph cluster deployed using Ceph-Ansible. We would like to migrate it to Rook. Are there any guides to help with this?

**Comment by travisn** [comment, marked as answer]
Two options to consider:
- Keep the ceph-ansible cluster and connect with the [external Rook cluster](https://rook.io/docs/rook/latest/CRDs/Cluster/external-cluster/)
- Backup the data and create a fresh Rook cluster where you would import the data.

**Reply by blakjak44** [reply]
We definitely don't want to keep the ceph-ansible deployment, but our data is nearly a PB so migration would be tricky. I'm not familiar with rook internals, but are the cluster components reading/writing metadata, keys, etc from the host? If so, is there a way to configure rook to point to the existing paths?

**Reply by travisn** [reply]
No, it's really just too complex of a migration to attempt, if it's even possible, and thus not supported.

**Reply by blakjak44** [reply]
Got it, thanks!
